src/mscorlib/src/System/Text/UTF8Encoding.cs

```diff
@@ -58,11 +58,13 @@ public class UTF8Encoding : Encoding
         internal sealed class UTF8EncodingSealed : UTF8Encoding
         {
             public UTF8EncodingSealed() : base(encoderShouldEmitUTF8Identifier: true) { }
```

Comment:
Do we still need this ctor, or could the s_default initialization just be `new UTF8EncodingSealed(encoderShouldEmitUTF8Identifier: true)?